userland/libtock/alarm.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <tock.h>
+
+typedef struct alarm alarm_t;
+
+alarm_t *alarm_start(uint32_t expiration, subscribe_cb, void*);
```

Comment:
Agree that alarm_at it's better. And should move the `alarm_read` function to this module (from the internal one)